packages/combobox/src/ComboboxMenu/Menu.styles.ts

```diff
@@ -109,7 +132,16 @@ export const menuMessageSizeStyle: Record<Size, string> = {
   `,
 };
```

Comment:
This object could also be a function if the typesScales.body1 reference was set in a const

Reply:
const getMenuMessageSizeStyle = (size: Size) =>  css`
    font-size: ${fontSize[size]}px;
    line-height: ${lineHeight[size]}px;
    padding: ${menuItemPadding[size].y}px
      ${menuItemPadding[size].x}px;
  `,